releasenotes/notes/prof-3-11-traceback-issue-20e06214d4a3879b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+issues:
+  - |
+    profiling: There is currently a known Python 3.11 compatibility issue where the stack collector does not properly 
+    access ``PyFrameObject`` member values, as ``PyFrameObject`` is now created and computed lazily in Python 3.11.
+    Until this is fixed, we advise against enabling the profiler while using Python 3.11.
```

Comment:
```suggestion
    Until this is fixed, we advise against enabling the profiler while using Python 3.11.
    
```
newline?

Reply:
No need for newline, it's just not showing up on github.